Compiling on Mac gets errors

**Describe the bug**
Following the README instructions for building nomacs on macOS, I get a build error during the "Run make" step.

**To Reproduce**
Steps to reproduce the behavior:
```
1. Clone the nomacs repository to your mac
2. Follow the directions for building nomacs for macOS at https://github.com/nomacs/nomacs#build-nomacs-macos
3. See error after the "Run make" step:
[  1%] Automatic MOC for target nomacsCore
[  1%] Built target nomacsCore_autogen
[  2%] Building CXX object CMakeFiles/nomacsCore.dir/nomacsCore_autogen/mocs_compilation.cpp.o
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/mocs_compilation.cpp:2:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/moc_qpsdplugin.cpp:10:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/../../../ImageLounge/3rdparty/libqpsd/qpsdplugin.h:25:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/QImageIOPlugin:1:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/qimageiohandler.h:43:
In file included from /opt/homebrew/include/QtGui/qtguiglobal.h:7:
/opt/homebrew/include/QtCore/qglobal.h:106:6: error: "Qt requires a C++17 compiler"
#    error "Qt requires a C++17 compiler"
     ^
/opt/homebrew/include/QtCore/qglobal.h:1009:26: error: no template named 'enable_if_t' in namespace 'std'; did you mean 'enable_if'?
         typename = std::enable_if_t<std::is_arithmetic_v<T> && std::is_arithmetic_v<U> &&
                    ~~~~~^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/type_traits:546:63: note: 'enable_if' declared here
template <bool, class _Tp = void> struct _LIBCPP_TEMPLATE_VIS enable_if {};
                                                              ^
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/mocs_compilation.cpp:2:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/moc_qpsdplugin.cpp:10:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/../../../ImageLounge/3rdparty/libqpsd/qpsdplugin.h:25:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/QImageIOPlugin:1:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/qimageiohandler.h:43:
In file included from /opt/homebrew/include/QtGui/qtguiglobal.h:7:
/opt/homebrew/include/QtCore/qglobal.h:1009:43: error: no template named 'is_arithmetic_v' in namespace 'std'; did you mean 'is_arithmetic'?
         typename = std::enable_if_t<std::is_arithmetic_v<T> && std::is_arithmetic_v<U> &&
                                     ~~~~~^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/type_traits:1151:50: note: 'is_arithmetic' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS is_arithmetic
                                                 ^
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/mocs_compilation.cpp:2:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/moc_qpsdplugin.cpp:10:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/../../../ImageLounge/3rdparty/libqpsd/qpsdplugin.h:25:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/QImageIOPlugin:1:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/qimageiohandler.h:43:
In file included from /opt/homebrew/include/QtGui/qtguiglobal.h:7:
/opt/homebrew/include/QtCore/qglobal.h:1009:70: error: no template named 'is_arithmetic_v' in namespace 'std'; did you mean 'is_arithmetic'?
         typename = std::enable_if_t<std::is_arithmetic_v<T> && std::is_arithmetic_v<U> &&
                                                                ~~~~~^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/type_traits:1151:50: note: 'is_arithmetic' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS is_arithmetic
                                                 ^
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/mocs_compilation.cpp:2:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/moc_qpsdplugin.cpp:10:
In file included from /Users/petr/Documents/GitHub/nomacs/build/nomacsCore_autogen/T3Q3FTK3DB/../../../ImageLounge/3rdparty/libqpsd/qpsdplugin.h:25:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/QImageIOPlugin:1:
In file included from /opt/homebrew/opt/qt5/lib/QtGui.framework/Headers/qimageiohandler.h:43:
In file included from /opt/homebrew/include/QtGui/qtguiglobal.h:7:
/opt/homebrew/include/QtCore/qglobal.h:1009:87: error: expected '>'
         typename = std::enable_if_t<std::is_arithmetic_v<T> && std::is_arithmetic_v<U> &&
                                                                                      ^
/opt/homebrew/include/QtCore/qglobal.h:1009:37: note: to match this '<'
         typename = std::enable_if_t<std::is_arithmetic_v<T> && std::is_arithmetic_v<U> &&
                                    ^
/opt/homebrew/include/QtCore/qglobal.h:1009:65: error: expected ',' or '>' in template-parameter-list
         typename = std::enable_if_t<std::is_arithmetic_v<T> && std::is_arithmetic_v<U> &&
                                                                ^
/opt/homebrew/include/QtCore/qglobal.h:1010:66: error: expected unqualified-id
                                     std::is_floating_point_v<T> == std::is_floating_point_v<U> &&
                                                                 ^
/opt/homebrew/include/QtCore/qglobal.h:1021:35: error: no type named 'Promoted' in namespace 'QTypeTraits::detail'
using Promoted = typename detail::Promoted<T, U>::type;
                 ~~~~~~~~~~~~~~~~~^~~~~~~~
/opt/homebrew/include/QtCore/qglobal.h:1021:43: error: expected ';' after alias declaration
using Promoted = typename detail::Promoted<T, U>::type;
                                          ^
/opt/homebrew/include/QtCore/qglobal.h:1036:31: error: no template named 'Promoted' in namespace 'QTypeTraits'
constexpr inline QTypeTraits::Promoted<T, U> qMin(const T &a, const U &b)
                 ~~~~~~~~~~~~~^
/opt/homebrew/include/QtCore/qglobal.h:1038:28: error: no template named 'Promoted' in namespace 'QTypeTraits'
    using P = QTypeTraits::Promoted<T, U>;
              ~~~~~~~~~~~~~^
/opt/homebrew/include/QtCore/qglobal.h:1039:5: error: unknown type name 'P'
    P _a = a;
    ^
/opt/homebrew/include/QtCore/qglobal.h:1040:5: error: unknown type name 'P'
    P _b = b;
    ^
/opt/homebrew/include/QtCore/qglobal.h:1044:31: error: no template named 'Promoted' in namespace 'QTypeTraits'
constexpr inline QTypeTraits::Promoted<T, U> qMax(const T &a, const U &b)
                 ~~~~~~~~~~~~~^
/opt/homebrew/include/QtCore/qglobal.h:1046:28: error: no template named 'Promoted' in namespace 'QTypeTraits'
    using P = QTypeTraits::Promoted<T, U>;
              ~~~~~~~~~~~~~^
/opt/homebrew/include/QtCore/qglobal.h:1047:5: error: unknown type name 'P'
    P _a = a;
    ^
/opt/homebrew/include/QtCore/qglobal.h:1048:5: error: unknown type name 'P'
    P _b = b;
    ^
/opt/homebrew/include/QtCore/qglobal.h:1052:31: error: no template named 'Promoted' in namespace 'QTypeTraits'
constexpr inline QTypeTraits::Promoted<T, U> qBound(const T &min, const U &val, const T &max)
                 ~~~~~~~~~~~~~^
/opt/homebrew/include/QtCore/qglobal.h:1058:31: error: no template named 'Promoted' in namespace 'QTypeTraits'
constexpr inline QTypeTraits::Promoted<T, U> qBound(const T &min, const T &val, const U &max)
                 ~~~~~~~~~~~~~^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[2]: *** [CMakeFiles/nomacsCore.dir/nomacsCore_autogen/mocs_compilation.cpp.o] Error 1
make[1]: *** [CMakeFiles/nomacsCore.dir/all] Error 2
make: *** [all] Error 2
```
**Expected behavior**
I expected no errors during build.

**Screenshots**
No screenshots.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version: Ventura 13.0.1

--- a/README.md
+++ b/README.md
@@ -111,12 +111,12 @@ $ cd build
 
 For Homebrew on Intel models:
 ```console
-$ Qt5_DIR=/usr/local/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/usr/local/opt/qt5/bin ../ImageLounge/.
+$ Qt5_DIR=/usr/local/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/usr/local/opt/qt5/bin/qmake ../ImageLounge/.
 ```
 
 For Homebrew on Apple Silicon models:
 ```console
-$ Qt5_DIR=/opt/homebrew/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/opt/homebrew/opt/qt5/bin ../ImageLounge/.
+$ Qt5_DIR=/opt/homebrew/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/opt/homebrew/opt/qt5/bin/qmake ../ImageLounge/.
 ```
 
 Run make:

